libs/nx-vite/src/executors/build/executor.ts

```diff
@@ -14,7 +15,10 @@ export default async function runExecutor(
     const appRoot = context.workspace.projects[context.projectName].root
 
     await build({
-        root: tree.root + '/' + appRoot,
+        root: join(tree.root, appRoot),
+        build: {
+            outputDir: join(tree.root, ('dist/apps/' + context.projectName)),
```

Comment:
Can we use `join(tree.root, ('dist/' + appRoot + context.projectName)),` just in case the app root is configured differently for the project